tools/run-tests.py

```diff
@@ -99,11 +99,11 @@ def run(args, cwd=None, env=None, stdout=None, checkresult=True, report=False):
 def readfile(filename):
     with open(filename, 'r') as f:
         return f.readlines()
-    
-def _run_wast_tests(engine, files, is_fail):
+
+def _run_wast_tests(engine, files, is_fail, jit=''):
     fails = 0
     for file in files:
-        proc = Popen([engine, file], stdout=PIPE)
+        proc = Popen([engine, file], stdout=PIPE) if jit == '' else Popen([engine, file, jit], stdout=PIPE)
```

Comment:
If you put --jit at the end it does not work, since it only affects files after it